components/edit-collective/sections/PaymentMethods.js

```diff
@@ -211,14 +286,55 @@ class EditPaymentMethods extends React.Component {
     }
   }
 
+  dismissSuccess = () => {
+    const { router } = this.props;
+    if (this.props.router.query.successType) {
+      const pathname = router.asPath.split('?')[0];
+      this.props.router.replace({ pathname, query: null }, undefined, { shallow: true });
+    }
+  };
+
   render() {
-    const { Collective, loading } = this.props.data;
+    const { account, loading } = this.props.data;
     const { showCreditCardForm, error, submitting, removedId, savingId } = this.state;
+    const successType = this.props.router.query.successType;
     const paymentMethods = this.getPaymentMethodsToDisplay();
+
     return loading ? (
       <Loading />
     ) : (
-      <Flex className="EditPaymentMethods" flexDirection="column">
+      <Flex className="EditPaymentMethods" flexDirection="column" pt={2}>
+        {this.hasOrdersThatNeedConfirmation() && (
+          <MessageBox type="warning" withIcon mb={3}>
+            <FormattedMessage defaultMessage="You need to confirm at least one of your payment methods." />
+          </MessageBox>
+        )}
+        {successType && (
+          <MessageBox
+            type="success"
+            display="flex"
+            alignItems="center"
+            withIcon
+            mb={3}
+            onClick={() => this.setState({ successType: null })}
+          >
+            <Flex justifyContent="space-between" alignItems="center" flex="1 1">
+              <span>
+                {successType === 'payment' ? (
+                  <FormattedMessage
+                    id="Order.Confirm.Success"
+                    defaultMessage="Your payment method has now been confirmed and the payment successfully went through."
+                  />
+                ) : (
+                  <FormattedMessage defaultMessage="Your payment method has been successfully added." />
+                )}
+              </span>
+              <StyledButton buttonSize="tiny" onClick={this.dismissSuccess}>
+                <FormattedMessage defaultMessage="Dismiss" />
+              </StyledButton>
```

Comment:
Should we use the ✖️ button?

Reply:
This is how it appears in the design:

![image](https://user-images.githubusercontent.com/1556356/210054772-ac706b77-89ac-477d-9786-bb48c8ac0e09.png)